guides/source/association_basics.md

```diff
@@ -7,20 +7,58 @@ This guide covers the association features of Active Record.
 
 After reading this guide, you will know how to:
 
+* Understand the various types of associations.
 * Declare associations between Active Record models.
-* Understand the various types of Active Record associations.
-* Use the methods added to your models by creating associations.
+* Choose the right association type for your models.
+* Use Single Table Inheritance.
+* Setting up and using Delegated Types.
 
 --------------------------------------------------------------------------------
 
-Why Associations?
+Associations Overview
 -----------------
 
-In Rails, an _association_ is a connection between two Active Record models. Why do we need associations between models? Because they make common operations simpler and easier in your code.
+_Associations_ in Rails are special macro style calls that make it easy to
+connect different models. You can use them to tell Rails how your models relate
+to each other, which helps you manage your data more effectively, and makes
+common operations simpler and easier to read.
```

Comment:
Final nitpick...

The Migrations, Callbacks and Active Record Basics guides talk about "Active Record Migration", "Active Record Callbacks" and "Active Record Associations" instead of "Rails Associations". Rails doesn't support other types of associations but would it be clearer to stick to "Active Record Associations" here as well?

Also can we avoid "macro style calls" in the opening sentence? 😄

Maybe something like:
```suggestion
Active Record associations allow you to define relationships between models.
_Associations_ are implemented as special macro style calls that make it easy to tell Rails how your models relate
to each other, which helps you manage your data more effectively, and makes
common operations simpler and easier to read.
```